.golangci.yml

```
@@ -0,0 +1,192 @@
+  modules-download-mode: readonly
+
+linters:
+  default: none
+  enable:
+    # Enabled by default
+    - errcheck
+    - govet
+    - ineffassign
+    - staticcheck
+    - unused
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)